cmd/status.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/codegangsta/cli"
+	"github.com/exercism/cli/api"
+	"github.com/exercism/cli/config"
+)
+
+// Status command allows a user to view their progress in a given language track.
+func Status(ctx *cli.Context) {
+	c, err := config.New(ctx.GlobalString("config"))
+	if err != nil {
+		log.Fatal(err)
+	}
+	args := ctx.Args()
+
+	if len(args) != 1 {
+		msg := "Usage: exercism status LANGUAGE"
+		log.Fatal(msg)
+	}
+
+	language := args[0]
+	client := api.NewClient(c)
+	status, err := client.Status(language)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	printResponse(status)
+}
+
+func printResponse(status *api.StatusInfo) {
```

Comment:
Still need to move this functionality from this standalone function to a method on StatusInfo. My major question, though, is where I should define that method - it can't be defined here because the type is defined in the `api` package, but I feel like adding it to the `api.go` file will be clutter. Should I maybe create a separate file for the type and endpoint processing of that particular command?